src/components/screens/device-models-screen.tsx

```diff
@@ -154,17 +153,19 @@ export class DeviceModelsScreen extends React.Component<Props, State> {
    */
   private renderDeviceModelsCardsList = () => {
     const { deviceModels } = this.props;
-    const cards = deviceModels.map((deviceModel) => {
-      return (
-        <CardItem
-          key={deviceModel.id}
-          title={`${deviceModel.manufacturer} ${deviceModel.model}`}
-          onClick={() => this.onCardClick(deviceModel)}
-          menuOptions={this.getCardMenuOptions(deviceModel)}
-          status={""}
-        />
-      );
-    });
+    const cards = [...deviceModels]
+      .sort((a, b) => a.manufacturer.localeCompare(b.manufacturer))
+      .map((deviceModel) => {
+        return (
+          <CardItem
+            key={deviceModel.id}
+            title={`${deviceModel.manufacturer} ${deviceModel.model}`}
+            onClick={() => this.onCardClick(deviceModel)}
+            menuOptions={this.getCardMenuOptions(deviceModel)}
+            status={""}
```

Comment:
Can this prop just be removed, its optional and is typed "small" | "large"

Reply:
Resolved.